lib/commands/setViewportSize.js

```diff
@@ -24,11 +24,10 @@
  *
```

Comment:
what I meant if indentation is here ^^^ could you also fix this (see line 8 - 14)